README.md

```diff
@@ -21,7 +21,7 @@ Each folder should contain a list of multiple intents, consider if the set of tr
 
 2. Create a new file with a folder title and a `NLU.md` file, or contribute to an existing folder.  
 
-3. In the `NLU.md` file, format your training data using markdown, and title each section with the intent types e.g.`intent:mood_happy` or `intent:mood_unhappy`
+3. In the `NLU.md` file, format your training data using markdown, remove all entities (see [script](https://github.com/RasaHQ/NLU-training-data#Why-do-I-need-to-remove-entities-from-my-training-data)), title each section with the intent types and add a short description e.g.`intent:mood_happy <!--expressing happy mood-->` or `intent:mood_unhappy <!--expressing unhappy mood-->`
```

Comment:
I'd give different examples for the descriptions. E.g.,
```
* intent:inform_rain
<!--
The user says that it is currently raining somewhere.
-->
```
since we want the descriptions to be meaningful and not just repeat the intent name. Here the "currently" and "somewhere" are important, since the define the "boundaries" of this intent.